components/identifier-generator/front/src/feature/tabs/SelectionTab.tsx

```diff
@@ -60,28 +66,30 @@ const SelectionTab: React.FC<SelectionTabProps> = ({target, conditions, onChange
         <Table.Body>
           {isLoading && <ListSkeleton />}
           {!isLoading && (
-            <>
-              <Table.Row>
-                <Styled.TitleCell>
-                  {identifiers && identifiers.length > 0 ? identifiers[0].label : `[${target}]`}
-                </Styled.TitleCell>
-                <Table.Cell colSpan={2}>
-                  <Styled.InputContainer>
-                    <TextInput value={translate('pim_common.operators.EMPTY')} readOnly={true} />
-                  </Styled.InputContainer>
-                </Table.Cell>
-              </Table.Row>
-              {conditionsWithId.map(conditionWithId => (
-                <>
-                  {conditionWithId.type === CONDITION_NAMES.ENABLED && (
-                    <EnabledLine condition={conditionWithId} key={conditionWithId.id} onChange={handleChange} />
-                  )}
-                </>
-              ))}
-            </>
+            <Table.Row>
+              <Styled.NotDraggableCell></Styled.NotDraggableCell>
```

Comment:
```suggestion
              <Styled.NotDraggableCell/>
```
There is no need to have both opening and closing tags when there's no children